spec/src/main/asciidoc/core/scopescontexts.asciidoc

```diff
@@ -228,6 +228,34 @@ From time to time, the container must obtain an _active context object_ for a ce
 
 If there is exactly one active instance of `Context` associated with the scope type, we say that the scope is _active_.
 
+==== Activating Built In Contexts
+
+Certain built in contexts support the ability to be activated and deactivated.  This allows developers to control built-in contexts in ways that they could also manage custom built contexts.
```

Comment:
I would start with something like this in order to emphasize that you normally don't do this, but you can should you need it:
`Built-in context activation and deactivation is managed by CDI container but sometimes it might be necessary to handle this manually. Certain contexts have this capability and therefore allows developers to control built-in contexts in ways that they could also manage custom built contexts.`
WDYT? Just wanted to point it out so that we avoid people (especially in SE I guess) who would overuse this.